Tests/MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -254,6 +254,7 @@ class CarPlayManagerTests: TestCase {
         
         let carPlayManagerDelegateMock = CarPlayManagerDelegateMock()
         let routeOptions = RouteOptions(coordinates: [
```

Comment:
Shall we also add a simple test that makes sure that we fail if one coordinate is provided?